src/content/docs/es/reference/error-reference.mdx

```diff
@@ -68,7 +63,9 @@ La siguiente referencia es una lista completa de los errores que puedes encontra
 - [**FailedToFindPageMapSSR**](/es/reference/errors/failed-to-find-page-map-ssr/)<br/>Astro couldn't find the correct page to render
 - [**MissingLocale**](/es/reference/errors/missing-locale/)<br/>The provided locale does not exist.
 - [**MissingIndexForInternationalization**](/es/reference/errors/missing-index-for-internationalization/)<br/>Index page not found.
+- [**IncorrectStrategyForI18n**](/es/reference/errors/incorrect-strategy-for-i18n/)<br/> You can't use the current function with the current strategy
 - [**NoPrerenderedRoutesWithDomains**](/es/reference/errors/no-prerendered-routes-with-domains/)<br/>Prerendered routes aren't supported when internationalization domains are enabled.
+- [**MissingMiddlewareForInternationalization**](/es/reference/errors/missing-middleware-for-internationalization/)<br/>Enabled manual internationalization routing without having a middleware.
```

Comment:
This page doesn't exist in spanish. Should I leave the link as it is?

Reply:
Hay que dejarlo por que falta traducir el error, lo marca el tracker.